src/client/videostreaming/StreamSelectorWrapper.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useContext } from "react";
+import { Container, Row, Button } from "react-bootstrap";
+import { FocusedVideoView } from "./FocusedVideoView";
+import { PeerContext } from "../peer";
+import {
+    turnAudioOff,
+    turnAudioOn,
+    turnVideoOff,
+    turnVideoOn,
+} from "../hooks/useMediaStream";
+
+export const StreamSelectorWrapper: React.FunctionComponent<{}> = () => {
+    const { stream: myStream, enableStream } = useContext(PeerContext);
+    return (
+        <Container>
+            <Row>
+                <FocusedVideoView />
+            </Row>
+            <Row>
+                <Button
+                    onClick={() => {
+                        if (!myStream) {
+                            enableStream();
+                        } else {
+                            turnAudioOn(myStream);
+                        }
+                    }}
+                >
+                    Unmute
+                </Button>
+                <Button
+                    onClick={() => {
+                        if (myStream) {
+                            turnAudioOff(myStream);
+                        }
+                        // socket.emit(VideoEvent.USER_STOP_STREAMING, myPeerId);
```

Comment:
^

Reply:
As specified in the description, StreamSelectorWrapper is supposed to be deleted by Tony when he understands how to use the video components.